src/VisualStudio/Core/Def/Implementation/ProjectSystem/AbstractProject.cs

```diff
@@ -1436,5 +1438,91 @@ protected static bool FilterException(Exception e)
             // Nothing fancy, so don't catch
             return false;
         }
+
+        #region FolderNames
```

Comment:
But didn't we want to leave this stuff in the AbstractLegacyProject? I can't imagine we'd want to use any of this for CPS projects?